processing/src/main/java/org/apache/carbondata/processing/util/CarbonDataProcessorUtil.java

```diff
@@ -424,6 +440,38 @@ public static boolean isHeaderValid(String tableName, String[] csvHeader,
     return noDicSortColMapping;
   }
 
+  /**
+   * Get the sort/no_sort column map based on schema order.
+   * This will be used in the final sort step to find the index of sort column, to compare the
+   * intermediate row data based on schema.
+   */
+  public static Map<Integer, List<Boolean>> getSortColSchemaOrderMapping(CarbonTable carbonTable) {
```

Comment:
Please add unit test for all the new methods

Reply:
added